lib/utilities/collectResponseHeaders.js

```diff
@@ -8,7 +8,8 @@
 function collectResponseHeaders(objectMD) {
     // Add user meta headers from objectMD
     const responseMetaHeaders = {};
-    Object.keys(objectMD).filter(val => val.substr(0, 11) === 'x-amz-meta-')
+    Object.keys(objectMD).filter(val => (val.substr(0, 11) === 'x-amz-meta-' ||
```

Comment:
I think foo.startsWith is more readable, although foo.substr works fine and might have some negligible performance edge. I just modified these functions to add the website header, so I didn't touch any other parts.